content/src/service/ServiceImpl.ts

```diff
@@ -81,12 +84,18 @@ export class ServiceImpl implements MetaverseContentService, TimeKeepingService,
     }
 
     deployEntity(files: ContentFile[], entityId: EntityId, auditInfo: AuditInfo, origin: string): Promise<Timestamp> {
-        return this.deployInternal(files, entityId, auditInfo, this.nameKeeper.getServerName(), ValidationContext.LOCAL, origin)
+        if (!this.allowDeploymentsWhenNotInDAO && !this.identityProvider.getOwnIdentity()) {
```

Comment:
I'm not sure if it's clear enough that having something in `this.identityProvider.getOwnIdentity()` means you're inside the DAO.